source/index.html

```diff
@@ -251,9 +252,7 @@ <h2>Totals</h2>
                     <h2>Weekly Overview</h2>
                 </header>
                 <div class="stats-flex">
-                    <figure class="graph">
-                        <canvas id="weekly-graph" height="180"></canvas>
-                    </figure>
+                    <div id="graph" class="weekly-graph"></div>
```

Comment:
See if you can use "figure" for this element instead of div, if possible it would convey the semantics more appropriately